test/client/fixtures/sandbox/code-instrumentation/getters-test.js

```diff
@@ -251,6 +251,23 @@ test('clean up outerHTML', function () {
     }
 });
 
+asyncTest('xhr.responseURL', function () {
+    var xhr   = new XMLHttpRequest();
+
+    xhr.addEventListener('readystatechange', function () {
+        if (this.readyState === XMLHttpRequest.DONE) {
+            if (this.responseURL)
+                strictEqual(eval(processScript('this.responseURL')), 'https://example.com/xhr-test/');
+            else
+                expect(0);
+
+            start();
+        }
+    });
+    xhr.open('GET', '/xhr-test/');
```

Comment:
You got 404 error.
Change url to `/xhr-test/100` because parameter `:delay` from 
https://github.com/DevExpress/testcafe-hammerhead/blob/master/test/client/config-qunit-server-app.js#L42 is required.